src/tools/clang.lua

```diff
@@ -226,9 +226,7 @@
 			x86 = "-m32",
 			x86_64 = "-m64",
 		},
-		flags = {
-			LinkTimeOptimization = "-flto",
-		},
+		linktimeoptimization = gcc.shared.linktimeoptimization,
```

Comment:
This line should also be added to `clang.shared` right?

Reply:
Good catch. I'll update that in a few.